tools/clang/tools/dxc/dxc.cpp

```diff
@@ -998,9 +978,9 @@ HRESULT DxcContext::FindModuleBlob(hlsl::DxilFourCC fourCC, IDxcBlob *pSource, I
   return E_INVALIDARG;
 }
 
-// This function is currently only supported on Windows due to usage of
-// IDiaTable.
-#ifdef LLVM_ON_WIN32
+// This function is currently only supported on Windows due to usage of IDiaTable.
+#ifdef _WIN32
+// TODO : There is an identical code in DxaContext in Dxa.cpp. Refactor this function.
```

Comment:
This comment is in the original dxc.cpp. It was accidentally removed. I'm bringing it back.